dkist/dataset/tests/test_dataset.py

```diff
@@ -51,6 +51,11 @@ def test_repr(dataset, dataset_3d):
     assert str(dataset_3d.data) in r
 
 
+@pytest.mark.accept_cli_dataset
+def test_flat_repr(large_tiled_dataset):
+    r = repr(large_tiled_dataset.flat)
```

Comment:
Can we add some kind of assert here? Is there a shape in the repr we can assert?

Reply:
Yeah the repr will include the length of the flat array so we can assert that. I'll get to it tomorrow.